lib/src/react_client/synthetic_event_wrappers.dart

```diff
@@ -0,0 +1,115 @@
+/// JS interop classes for React synthetic events.
+///
+/// For use in `react_client.dart` and by advanced react-dart users.
+library react_client.synthetic_event_wrappers;
+
+import 'dart:html';
+import 'package:js/js.dart';
+
+@JS()
+@anonymous
+class SyntheticEvent {
+  external bool get bubbles;
+  external bool get cancelable;
+  external get currentTarget;
+  external bool get defaultPrevented;
+  external num get eventPhase;
+  external bool get isTrusted;
+  external get nativeEvent;
+  external get target;
+  external num get timeStamp;
+  external String get type;
+
+  external void stopPropagation();
+  external void preventDefault();
+
+  external void persist();
```

Comment:
Woah, it is documented now! See the callout here: https://facebook.github.io/react/docs/events.html#event-pooling